### PR TITLE
Feature calculate total hours

### PR DESCRIPTION
**Pull request by lhohenrique**

### CODE REVIEW

File: PontoFacil/PontoFacil/Services/Interfaces/ISettingsService.cs [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+using PontoFacil.Models;
 
 namespace PontoFacil.Services.Interfaces
 {
     public interface ISettingsService
     {
         void Save(Profile profile);
         Profile GetProfile();
+        void UpdateProfileAcumulatedHours(TimeSpan overtimeHours);
+        string GetProfileAccumulatedHours();
```
Comment (thiago-rocha) [inline PontoFacil/PontoFacil/Services/Interfaces/ISettingsService.cs:11]: These two lines should be in a new Interface, (e.g. `IEditProfle`)

**Review by thiago-rocha — COMMENTED** [review]
@lhohenrique avoid commits names like "Minor Changes", "Partial something", Try be more specific like "Change button style on History Page"

File: PontoFacil/PontoFacil/ViewModels/HistoryPageViewModel.cs [hunk 2]
```diff
@@ -41,11 +49,11 @@ public ObservableCollection<ClockIn> History
         #endregion
 
         #region Constructor
-        public HistoryPageViewModel(IHistoryService historyService)
+        public HistoryPageViewModel(IHistoryService historyService, ISettingsService settingsService)
         {
             InitializeCommands();
 
-            _historyService = historyService;
+            _historyService = historyService;
```
Comment (thiago-rocha) [inline PontoFacil/PontoFacil/ViewModels/HistoryPageViewModel.cs]: Remove tab

File: PontoFacil/PontoFacil/ViewModels/HistoryPageViewModel.cs [hunk 3]
```diff
@@ -33,6 +34,13 @@ public ObservableCollection<ClockIn> History
             set { SetProperty(ref _history, value); }
         }
 
+        private string _accumulatedHours;
+        public string AccumuletedHours
```
Comment (thiago-rocha) [inline PontoFacil/PontoFacil/ViewModels/HistoryPageViewModel.cs]: Accumul **a** tedHours

**Review by diego-millan — APPROVED** [review]

**Pull request merged**